domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt

```diff
@@ -532,6 +532,32 @@ class ProfileManagementControllerTest {
     val profile = monitorFactory.waitForNextSuccessfulResult(profileProvider)
     assertThat(profileManagementController.getCurrentProfileId().internalId).isEqualTo(2)
     assertThat(profile.lastLoggedInTimestampMs).isNotEqualTo(0)
+    assertThat(profile.numberOfLogins).isNotEqualTo(0)
```

Comment:
Shouldn't numberOfLogins be one if the profile was logged in?

I'd think we'd want to check the profile before and after it's logged in to verify the initial login count state, and updated login count state.

Reply:
done